contracts/upgradeable_contracts/erc20_to_erc20/HomeBridgeErcToErc.sol

```diff
@@ -27,6 +29,78 @@ contract HomeBridgeErcToErc is ERC677Receiver, EternalStorage, BasicBridge, Basi
         address _owner
     ) public
       returns(bool)
+    {
+        _initialize (
+            _validatorContract,
+            _dailyLimit,
+            _maxPerTx,
+            _minPerTx,
+            _homeGasPrice,
+            _requiredBlockConfirmations,
+            _erc677token,
+            _foreignDailyLimit,
+            _foreignMaxPerTx,
+            _owner
+        );
+        setInitialize(true);
+
+        return isInitialized();
+    }
+
+    function rewardableInitialize (
+        address _validatorContract,
+        uint256 _dailyLimit,
+        uint256 _maxPerTx,
+        uint256 _minPerTx,
+        uint256 _homeGasPrice,
+        uint256 _requiredBlockConfirmations,
+        address _erc677token,
+        uint256 _foreignDailyLimit,
+        uint256 _foreignMaxPerTx,
+        address _owner,
+        address _blockReward,
```

Comment:
Added `POSDAOHomeBridgeErcToErc` here https://github.com/poanetwork/poa-bridge-contracts/pull/166/commits/8ab76b90445420da077ad096f01a2ae68cd5b7d7. 

> Another idea is that blockRewardContract and setBlockRewardContract should be implemented in the fee manager since they are related to the fee distribution functionality. In this case the bridge contract should proxy calls to the fee manager contract.

I implemented this approach, it makes sense to manage blockReward logic on fee manager since it is related to this contract functionality.